### PR TITLE
[AppBar] Updated accessibility label on back button

### PR DESCRIPTION
**Pull request by ShepJGoogle**
Change accessibility label on AppBar back button

### CODE REVIEW

File: components/AppBar/src/MDCAppBar.m [hunk 1]
```diff
@@ -220,6 +220,7 @@ - (UIBarButtonItem *)backButtonItem {
                                                         action:@selector(didTapBackButton:)];
   }
   backBarButtonItem.accessibilityIdentifier = @"back_bar_button";
+  backBarButtonItem.accessibilityLabel = NSLocalizedString(@"Back", @"Back");
```
Comment (Arcank) [inline components/AppBar/src/MDCAppBar.m]: Won’t you need a strings file in the AppBar component? Otherwise this will never be localized.

Comment (ShepJGoogle) [reply components/AppBar/src/MDCAppBar.m]: Yep I only published this as it is a blocker for a project that needed a temporary fix and this was a way for me to display the fix. I've marked it with a "isPlanningChanges" label so nobody merged it or reviewed it.

**Review by samnm — APPROVED** [review]

**Review by ianegordon — APPROVED** [review]

**Review by ianegordon — COMMENTED** [review]
Travis is missing a AppBarStrings.h file which seems like an actual issue.  Otherwise LGTM.
Can you take a look before submit?

**Comment by ShepJGoogle** [comment]
@ianegordon Checked in update to pod spec

**Review by ianegordon — CHANGES_REQUESTED** [review]
Waiting for Travis.

**Comment by ShepJGoogle** [comment]
@ianegordon The podspec was updated

**Comment by ianegordon** [comment]
I think that was a stale page issue. disregard.

**Review by ianegordon — APPROVED** [review]

**Comment by ianegordon** [comment]
Verified on an iPhone 6 - 10.1.1.

**Pull request merged**